### DISCUSSION TITLE
SqlServer 2019 AD Authentication with user name different from current

### DISCUSSION THREAD

**Discussion by michaelplavnik** [dotnet/SqlClient#2180] [Q&A]
Is there a way to authenticate against on premises SQL Server 2019 with Microsoft.Data.SqlClient (dotnet6 runtime, Windows 10) given AD domain user name and password? 

User name is different from the current user and was verified to authenticate against same server under impersonated context. However, impersonated context has some other challenges within dotnet6 runtime that prevents me from using it as a solution.

**Comment by David-Engel** [comment]
There is no way to pass an AD username and password to Microsoft.Data.SqlClient for use in authenticating against SQL Server.
